doc/devices/ahs_remote.rst

```diff
@@ -0,0 +1,100 @@
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed, and once you
+`sign up for Amazon Braket <https://docs.aws.amazon.com/braket/latest/developerguide/braket-enable-overview.html>`_,
+you have access to the remote AHS device in PennyLane.
+
+Instantiate an AWS device that communicates with the hardware like this:
+
+>>> import pennylane as qml
+>>> s3 = ("my-bucket", "my-prefix")
+>>> remote_device = qml.device("braket.aws.ahs", device_arn="arn:aws:braket:us-east-1::device/qpu/quera/Aquila", s3_destination_folder=s3, wires=3)
+
+This device can be used with a QNode within PennyLane. It accepts circuits with a single ``ParametrizedEvolution``
+operator based on a hardware-compatible ``ParametrizedHamiltonian``. More information about creating PennyLane operators for AHS can be
+found in the `PennyLane docs <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_.
+
+Creating a register
+^^^^^^^^^^^^^^^^^^^
+
+The atom register defines where the atoms will be located, and determines the strength of the interaction
+between the atoms. Here we define coordinates for the atoms to be placed at (in micrometers), and create a constant
+interaction term for the Hamiltonian:
+
+.. code-block:: python
+
+    coordinates = [[0, 0], [0, 5], [5, 0]]  # number of coordinate pairs must match number of device wires
+
+    H_interaction = qml.pulse.rydberg_interaction(coordinates)
+
+Creating a global drive
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Hardware currently supports a single global drive pulse applied to all atoms in the register.
+
+Here we define a global drive with time dependent amplitude and detuning, with phase set to 0.
+
+.. code-block:: python
+
+    # gaussian amplitude function (qml.pulse.rect enforces 0 at start and end for hardware)
+    def amp_fn(p, t):
+        f = p[0] * jnp.exp(-(t-p[1])**2/(2*p[2]**2))
+        return qml.pulse.rect(f, windows=[0.1, 1.7])(p, t)
+
+
+    # defining a linear detuning
+    def det_fn(p, t):
+        return p * t
+
+    # creating a global drive on all wires
+    H_global = qml.pulse.rydberg_drive(amplitude=amp_fn, phase=0, detuning=det_fn, wires=[0, 1, 2])
+
+Creating and executing the circuit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once we have the terms describing the atomic interactions and the electromagnetic drive on the atoms, we can create
+and execute a circuit to run the pulse program on the hardware:
+
+.. code-block:: python
+
+    @qml.qnode(remote_device)
+    def circuit(amp_params, detuning_params):
+        qml.evolve(H_interaction + H_global)([amp_params, det_params], t=1.75)
+        return qml.sample()
+
+When executed, the circuit performs the computation on the hardware.
+
+>>> amp_params = [2.5, 1, 0.3]  # amp_fn expects p to contain 3 parameters
+>>> det_params = 0.2  # det_fn expects p to be a single parameter
+>>> circuit(amp_params, det_params)
+array([0.97517033, 0.04904283])
```

Comment:
# TODO: update output! -- this example has been run and a hardware task created, will be done tomorrow

We might also want to update this to be `expval` once that is implemented